src/components/ListItem.js

```diff
@@ -38,12 +21,15 @@ const ListItem = (props) => {
     }
   }
   return (
-    <li onClick={() => onClickHandler({ id, task, completed, editTodo })}>
-      <div>{task}</div>
-      <div>
-        <button type='button' name='delete' onClick={() => remove(id)}>Delete Task</button>
-      </div>
-    </li>
+    <Link to={`/todos/${id}`}>
+      <li>
+        <div>{task}</div>
+        <div>
+          <button type='button' name='delete' onClick={() => remove(id)}>Delete Task</button>
```

Comment:
Not sure if this will be an issue, but you may want to check that a click on this element does not propagate up to the `Link` and cause navigation.